opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -1443,7 +1443,7 @@ File {} line {}.)", pattern, location.keyword, location.filename, location.linen
                                     errors,
                                     grid,
                                     matching_wells,
-                                    /*actionx_mode=*/false,
+                                    /*actionx_mode=*/true,
```

Comment:
Is this a bugfix or does the PR provide capabilities beyond what was possible at the introduction of this support in PR #2833 (commit 372b7e86a4)?  I notice that this change will affect the way we [`handleEXIT()`](https://github.com/OPM/opm-common/blob/5daf8492490941c51355f55bb4d1dae1dd2779ae/opm/input/eclipse/Schedule/KeywordHandlers.cpp#L110), but it will also affect the way we [`handleWELPI()`](https://github.com/OPM/opm-common/blob/5daf8492490941c51355f55bb4d1dae1dd2779ae/opm/input/eclipse/Schedule/Well/WellPropertiesKeywordHandlers.cpp#L195-L198).  Is the latter fully supported now?  If not, I think I'd prefer that we "just" remove the conditional in `handleEXIT()` instead of flipping this switch.

Reply:
handleWELPI is not fully supported yet - I  then EXIT can be tr